utility/sosreport.py

```diff
@@ -54,7 +54,7 @@ def generate_sosreport_in_node(
         ssh_d.connect(nodeip, username=uname, password=pword)
         ssh_d.exec_command("sudo yum -y install sos")
         stdin, stdout, stderr = ssh_d.exec_command(
-            "sudo sosreport -a --all-logs -e ceph --batch"
+            "sudo sos report -a --all-logs --batch"
```

Comment:
Is this something changed with RHEL 9? Wdyt about checking for the OS before issuing the command?

Reply:
This is working on RHEL 8 and is specific with SOS version.